hack/update-generated-crd.sh

```diff
@@ -10,23 +10,29 @@ function install_crd {
   fi
 }
 
+
 # Can't rely on associative arrays for old Bash versions (e.g. OSX)
-install_crd \
+
+[[ -e "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-Default.crd.yaml" ]] && install_crd \
   "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-Default.crd.yaml" \
   "manifests/00-custom-resource-definition.yaml"
 
-install_crd \
+[[ -e "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-CustomNoUpgrade.crd.yaml" ]] && install_crd \
   "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-CustomNoUpgrade.crd.yaml" \
   "manifests/00-custom-resource-definition-CustomNoUpgrade.yaml"
 
-install_crd \
+[[ -e "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-DevPreviewNoUpgrade.crd.yaml" ]] && install_crd \
   "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-DevPreviewNoUpgrade.crd.yaml" \
   "manifests/00-custom-resource-definition-DevPreviewNoUpgrade.yaml"
```

Comment:
I know I suggested adding the checks on the `install_crd` commands, but what do you think about putting the check in `install_crd` function itself?
```
function install_crd {
  local SRC="$1"
  local DST="$2"
  if [[ -e "$SRC" ]]
  then
    if [[ -e "$DST" ]]
    then
      if ! diff -Naup "$SRC" "$DST"; then
        cp "$SRC" "$DST"
        echo "updated CRD: $SRC => $DST"
      else
        echo "skipped CRD that is already up to date: $DST"
      fi
    else
      cp "$SRC" "$DST"
      echo "updated CRD: $SRC => $DST"
    fi
  else
    if [[ -e "$DST" ]]
    then
      rm "$DST"
      echo "removed CRD that not vendored: $DST"
    else
      echo "skipped CRD that is not vendored: $SRC"
    fi
  fi
}
```
Then you don't need to change the `install_crd` commands in this PR to repeat the check on each one.

Reply:
done.